src/main/scala/org/constellation/primitives/storage/Storage.scala

```diff
@@ -32,4 +32,8 @@ trait Storage[F[_], K, V] {
   def toMap(): F[Map[K, V]]
 
   def cacheSize(): Long
+
+  def getLast20: F[List[V]]
+
+  def getLast20Sync: List[V]
```

Comment:
Can `getLast` and its variants be a function and parametrized by the same value below, i.e.

```scala
private val maxQueueSize = 20

Reply:
it needs more refactoring and dynamic queue and if such feature will be needed, we'll do it separately